src/common/entity/cover_icon.ts

```diff
@@ -4,17 +4,58 @@ import { domainIcon } from "./domain_icon";
 
 export const coverIcon = (state: HassEntity): string => {
   const open = state.state !== "closed";
+
   switch (state.attributes.device_class) {
     case "garage":
-      return open ? "hass:garage-open" : "hass:garage";
+      switch (state.state) {
+        case "opening":
+          return "hass:arrow-up-bold-box";
```

Comment:
I think we shouldn't go for the `bold` variant to align with the style of the buttons

Reply:
Good call. Will adjust

Reply:
That took longer to test than I expected. It looks like I have to restart scripts/develop for it to see new icons

Reply:
Yes, it generates the `hass` icon set when you start `develop`